assets/app/scripts/services/applicationGenerator.js

```diff
@@ -231,8 +222,8 @@ angular.module("openshiftConsole")
           },
           source: {
             git: {
-              ref: sourceRef,
-              uri: sourceUrl
+              ref: input.buildConfig.gitRef || 'master',
```

Comment:
v2 actually allowed specifying the ref in either the URL with a # or by putting it in the separate ref input field, so that you could use the same "URL" in both the CLI and the web.  Its a bit weird I know.